File: solution/java/day23/src/main/java/org/craftedsw/tripservicekata/trip/TripDAO.java
package org.craftedsw.tripservicekata.trip;

import io.vavr.collection.Seq;
import org.craftedsw.tripservicekata.exception.CollaboratorCallException;
import org.craftedsw.tripservicekata.user.User;

public class TripDAO {

    public static Seq<Trip> findTripsByUser(User user) {
        throw new CollaboratorCallException(
                "TripDAO should not be invoked on an unit test.");
    }

}

File: solution/c#/Day23/Day23/Trips/TripDAO.cs
using Day23.Exception;
using Day23.Users;
using LanguageExt;

namespace Day23.Trips
{
    public static class TripDAO
    {
        public static Seq<Trip> FindTripsByUser(User user)
        {
            throw new CollaboratorCallException("TripDAO should not be invoked on an unit test.");
        }
    }
}